docker-image/Dockerfile

```
@@ -41,12 +42,24 @@ RUN apt-get update \
         docker-ce-cli \
         python3.7 \
         python3-pip \
+        rubygems \
 && apt-get clean \
 \
 && python3.7 -m pip install --upgrade \
         pip setuptools wheel \
 && python3.7 -m pip install --upgrade \
-        boto3 jinja2 pytest python-json-logger ruamel.yaml
+        boto3 \
+        jinja2 \
+        pytest \
+        python-json-logger \
+        ruamel.yaml \
+        pylint \
+        pylint-fail-under \
+        ansible-lint \
+        ansible-lint-junit==0.16 \
```

```suggestion
        ansible-lint \
        ansible-lint-junit>=0.16 \
        boto3 \
        jinja2 \
        pylint \
        pylint-fail-under \
        pytest \
        python-json-logger \
        ruamel.yaml \
```
Let's keep alphabetical order. The latest ansible-lint-junit in PyPI is 0.16 so I'm not sure if we need to set it. If yes, maybe >=0.16 would be better?